llx/builtin_ip.go

```diff
@@ -49,17 +62,10 @@ func NewIP(s string) IP {
 			mask = 64
 		}
 	}
-
-	return IP{
-		IP:      ip,
-		Version: version,
-		Mask:    mask,
-	}
+	return version, mask
 }
 
-var bitmasks = []byte{0x00, 0x80, 0xc0, 0xe0, 0xf0, 0xf8, 0xfc, 0xfe, 0xff}
-
-func int2ip(i int64) string {
+func int2ip[T int | int64 | int32 | uint | uint64 | uint32](i T) net.IP {
```

Comment:
beautiful ✨

Reply:
Just a question since I see in MQL we use `int64` everywhere. It looks awesome but I'm curious if we are even using all of those types? 🤔